### PR TITLE
Test fix for silent mode disabled

### PR DESCRIPTION
**Pull request by num8er**
fixed test when `purity.silent = false` throws error

### CODE REVIEW

**Comment by num8er** [comment]
@abbasudo  got error when in config silent set to false during restriction test so made a fix on test with turned off silent too.

another issue I solve during custom resolver feature, since OrFilter calls Resolve::class (which is correct) so made sure model is resolvable if requested through  Resolve::class

**Comment by abbasudo** [comment]
`OrFilter` should be calling the replaced resolver class. No need for `defaultFilterResolverClass`.

File: src/Traits/Filterable.php [hunk 1]
```diff
@@ -79,6 +81,7 @@ private function bootFilter(): void
             return (new FilterList())->only($this->getFilters());
         });
 
+        app()->when($this->defaultFilterResolverClass)->needs(Model::class)->give(fn () => $this);
```
Comment (abbasudo) [inline src/Traits/Filterable.php:84]: There should be one single resolver class during the lifecycle of any request. remove this line and update `OrFilter` to accept the replaced class.

**Review by abbasudo — CHANGES_REQUESTED** [review]

File: src/Traits/Filterable.php (hunk 1 above)
Comment (num8er) [reply src/Traits/Filterable.php:84]: Seems like I'll be updating it during the day.
I wanted to keep it simple.
But agree with You.

Comment (num8er) [reply src/Traits/Filterable.php]: @abbasudo I'll make separate PR for it.

**Comment by abbasudo** [comment]
`defaultFilterResolverClass` variable is redundant and should be removed.

**Pull request merged**